Add EmCasa

gusaiani opened the pull request:
Hi all,

here's EmCasa, a Real Estate company in Brazil.

Cheers!

———

Please review our [Contribution Guidelines](https://github.com/doomspork/elixir-companies/blob/master/CONTRIBUTING.md) to ensure your changes abides by it.

Review your changes and complete the checklist below.

- [x] Company name
- [x] Company description
- [x] Company location
- [x] GitHub link
- [x] How Elixir is used
- [x] Sorted alphabetically

stevedomin commented:
Thanks @gusaiani!

Just fyi looks your SSL configuration is broken.

stevedomin commented:
Sorry, just one thing before I merge, can you add the location at the end just so that it's consistent.

Something like "[...]. Rio de Janeiro, Brazil."

gusaiani commented:
Thanks for pointing the https situation out, @stevedomin. Avoid Heroku + Google Domains if you can :)

I removed `https` from the link and added location at the end.

Thank you very much.
